__tests__/app.test.js

```diff
@@ -86,6 +87,38 @@ describe('/api/categories', () => {
                     expect(body[4].comment_count).toBe(3)
                 })
             })
+            test('Optional parameters - sort_by', () => {
+                return request(app).get('/api/reviews?sort_by=votes').expect(200).then(({body}) => {
+                    expect(body).toBeSortedBy('votes', {descending: true})
+                })
+            })
+            test('Optional parameters - order', () => {
+                return request(app).get('/api/reviews?order=asc').expect(200).then(({body}) => {
+                    expect(body).toBeSortedBy('created_at')
+                })
+            })
+            test('Optional parameters - category', () => {
+                return request(app).get('/api/reviews?category=euro%20game').expect(200).then(({body}) => {
+                    body.forEach(game => {
+                        expect(game.category).toBe('euro game')
+                    })
+                })
+            })
+            test('Optional parameters - sort_by - invalid input', () => {
+                return request(app).get('/api/reviews?sort_by=beans').expect(200).then(({body}) => {
+                    expect(body).toBeSortedBy('created_at', {descending: true})
+                })
+            })
+            test('Optional parameters - order - invalid input', () => {
+                return request(app).get('/api/reviews?order=yes').expect(200).then(({body}) => {
+                    expect(body).toBeSortedBy('created_at', {descending: true})
+                })
+            })
+            test('Optional parameters - order - invalid but possible input', () => {
+                return request(app).get('/api/reviews?category=funnygames').expect(200).then(({body}) => {
+                    expect(body.length).toBe(0)
+                })
+            })
```

Comment:
Id like to see some tests for the default cases too (requests without the querys should be ordered by date descending)